### PR TITLE
generate sitemap.xml during prebuild, updated simtemap.xml

### PR DESCRIPTION
**Pull request by Palabola**
## Summary by CodeRabbit

- **New Features**
	- Introduced sitemap generation to improve site navigation and SEO capabilities.
	- Enhanced the processing script to create a comprehensive sitemap with static and dynamic URLs.

- **Dependencies**
	- Added `sitemap` library to support the new sitemap functionality.
	- Added `xml-formatter` library for improved XML formatting.

### CODE REVIEW

File: prebuild.js [hunk 1]
```diff
@@ -58,4 +58,66 @@ data = data.sort((a, b) => new Date(b.date) - new Date(a.date));
...
+            .replaceAll('</url>', '\n  </url>')
+            .replaceAll('<loc>', '\n    <loc>')
+            .replaceAll('<changefreq>', '\n    <changefreq>')
+            .replaceAll('<priority>', '\n    <priority>'));
+      }
+      )
+      //.then(xml2js.parseStringPromise)
+      //.then(obj => xmlbuilder.create(obj).end({ pretty: true }))
+      //.then(prettyXml => console.log(prettyXml))
+      .catch(err => console.error(err));
```
[the remark this answers is not in the document]
Comment (Palabola) [reply prebuild.js:114]: It failed to convert the XML this is why choose the homemade version

**Comment by daroczig** [comment]
Please make sure to use the previously decided priorities from https://sparecores.com/sitemap.xml, and also make sure that there are no pages missing, e.g. `/talks`.

File: prebuild.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (daroczig) [reply prebuild.js:114]: @Palabola could you please get back on this? after a quick test, I have not seen any errors with the `xml-formatter` approach

[the remark this answers is not in the document]
Comment (Palabola) [reply prebuild.js]: It works indeed, it seems I messed up something when trying this approach.

**Comment by daroczig** [comment]
@Palabola as per above:

> Please make sure to use the previously decided priorities from https://sparecores.com/sitemap.xml

And also cleanup the dead code around the XML formatter 🙏

**Review by daroczig — APPROVED** [review]

**Pull request merged**